test/api/sample_functions_test.cpp

```diff
@@ -0,0 +1,15 @@
+#include <gtest/gtest.h>
+
+#include <bamit/sample_functions.hpp>
+
+TEST(sample_functions_test, sample_read_depth_test)
```

Comment:
It is difficult to write a test for the result because it is not deterministic due to random sampling, right? Maybe you can make an example SAM where the read depth is always 2? Such that the result is always the same.

Reply:
Yes, but also because all of the statistics are stored as doubles, and doubles are basically impossible to compare for equality (from what I could see)...